scripts/build/gen_relocate_app.py

```diff
@@ -513,27 +519,31 @@ def parse_input_string(line):
         mem_region = mem_region.rstrip()
         phdr, rest = rest.split(':', 1)
 
+    flag_list, rest = rest.split(':', 1)
+
     # Split lists by semicolons, in part to support generator expressions
-    flag_list, file_list = (lst.split(';') for lst in rest.split(':', 1))
+    file_list, symbol_filter = (lst.split(';') for lst in rest.split(',', 1))
 
-    return mem_region, phdr, flag_list, file_list
+    return mem_region, phdr, [flag_list], file_list, symbol_filter
```

Comment:
`flag_list` needs to be split by `;` before returning. This used to happen in the previous line.

Reply:
Good point, will fix.